scala/scala-impl/src/org/jetbrains/sbt/project/structure/SbtStructureDump.scala

```diff
@@ -28,7 +28,7 @@ import scala.collection.JavaConverters._
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
-class SbtStructureDump {
+class SbtStructureDump extends ProjectDumper {
```

Comment:
"Dumper" maybe a it of a misnomer for mill, perhaps call it something like "PreImport"?